guides/common/modules/ref_supported-cloud-providers.adoc

```diff
@@ -0,0 +1,11 @@
+[id="supported-cloud-providers_{context}"]
+= Supported Cloud Providers
+
+You can connect {Project} using compute resources to the following cloud providers:
+
+ifndef::orcharhino[]
```

Comment:
Does Orcharhino not ship OpenStack? It's included in Foreman itself so this does surprise me.

Reply:
Yes, orcharhino does not officially support OpenStack: https://docs.orcharhino.com/or/docs/sources/compute_resources.html